README.md

```diff
@@ -47,34 +47,59 @@ The fields in the table below can be used in these parts of STAC documents:
 | proj:transform   | \[number]       | The affine transformation coefficients for the default grid  |
 
 At least one of the fields MUST be specified, but it is RECOMMENDED to provide more information as detailed in the
-[Best Practices section](#best-practices) so that, for example, GDAL can read your data without issues.
-
-Also, [version 1.0.0](https://github.com/stac-extensions/projection/releases/tag/v1.0.0) of this extension
-required `proj:epsg` (either as integer or null) in Item Properties.
-This is not required anymore, but it is still recommended to additionally provide an EPSG code whenever available.
+[Best Practices section](#best-practices) so that, for example, GDAL can read your data without issues. 
 
 Generally, it is preferrable to provide the projection information on the Asset level
 as they are specific to assets and may not apply to all assets.
 For example, if you provide a smaller and unlocated thumbnail, having the projection information in the Item Properties
 would imply that the projection information also applies to the thumbnail if not specified otherwise in the asset.
-You may want to add the EPSG code to the Item Properties though as this would provide an easy way to
-filter for specific EPSG codes in an API.
-In this case you could override the EPSG code for the thumbnail on the asset level.
+You may want to add the `proj:code` to the Item Properties though as this would provide an easy way to
+filter for specific projection codes in an API. In this case you could override the `proj:code` for the thumbnail on the asset level.
 
 ### Additional Field Information
 
 #### proj:epsg
 
-A Coordinate Reference System (CRS) is the data reference system (sometimes called a
-'projection') used by the asset data, and can usually be referenced using an 
-[EPSG code](https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset).
-A great tool to help find EPSG codes is [epsg.io](http://epsg.io/).
+**Notice**: This field has been removed in V2.0.0. See [proj:epsg Migration to V2](#projepsg-migration-to-projcode).
 
-This field SHOULD be set to `null` in the following cases:
+#### proj:epsg migration to proj:code
+`proj:epsg` is removed in version 2.0.0 of this extension. Please use `proj:code`. For example, the following:
+
+```json
+{
+  ...
+  "proj:epsg": 32659,
+  ...
+}
+```
+would be represented as:
+```json
+{
+  ...
+  "proj:code": "EPSG:32659",
+  ...
+}
+```
+
+#### proj:code
+
+Projection codes are identified by a string. The [proj](https://proj.org/) library defines projections 
+using "authority:code", e.g., "EPSG:4326" or "IAU_2015:30100". Different projection authorities may define 
+different string formats. Examples of known projection authorities, where when can find well known codes that 
+clients are likely to support are listed in the following table.
+
+| Authority Name | URL |
```

Comment:
@jlaura It looks like the linter wants the table to be formatted more like this:

```
| Authority Name                          | URL                                                        |
| --------------------------------------- | ---------------------------------------------------------- |
| European Petroleum Survey Groups (EPSG) | <http://www.opengis.net/def/crs/EPSG> or <http://epsg.org> |
| International Astronomical Union (IAU)  | <http://www.opengis.net/def/crs/IAU/2015>                  |
| Open Geospatial Consortium (OGC)        | <http://www.opengis.net/def/crs/OGC>                       |
```

Reply:
Great, tests are passing now!

Reply:
I think good to merge? 😰

Reply:
Yeah, would be good to have 1-2 more people to approve. Thus I tagged a couple of people above. Could also be others...

Reply:
I hate to ask, but if we're adding IAU, why aren't we also adding the Esri registry?

Reply:
Happy to add! Link?

Reply:
I don't know that anyone's really happy to add it 😄 

Case against Esri registry:
* No versioning afaik
* Release cadence is whenever they feel like it
* No public resource to point at

Reply:
One case for, maybe not a strong one, is that they are in the PROJ library and ship with things like QGIS (among a ton of others). So, if a STAC item declares say ESRI:53029, the client would know to use Sphere_Van_Griten_I just from the code.

I think though, that we discussed (?) codes being more informative and clients really wanting the WKT / proj.json so that might not be a strong rationale. (I have a vague recollection of a discussion on a PR perhaps?)

Reply:
The extension doesn't say which registries are allowed. There's nothing in the schema that prevents from using ESRI:xxxx codes. The table in the extension is more a heads-up for potential registries, whether we add or don't add ESRI doesn't make a huge difference here. So I'd say just add it.

Reply:
Here is a link to maybe include? https://spatialreference.org/ref/esri/

To Howard's point, I'm not sure of another link. I'll get this one added can change if someone has another link they's prefer.